test-crawler/extractor/extractor.go

```diff
@@ -163,9 +188,12 @@ func getMetadata(content string, treeCursor *sitter.TreeCursor, parser *a.Parser
 	return &meta
 }
 
+// this method extracts functions and methods.
+// Function can have 2 types: function_declaration (for example contructor)
+// and method_declaration (can be exported and unexported)
```

Comment:
Please format this comment in line with GoDoc standards, and add an explanation of the return value.